server/src/main/java/org/apache/iotdb/db/engine/StorageEngine.java

```diff
@@ -49,12 +49,12 @@
 import org.apache.iotdb.db.engine.flush.TsFileFlushPolicy;
 import org.apache.iotdb.db.engine.flush.TsFileFlushPolicy.DirectFlushPolicy;
 import org.apache.iotdb.db.engine.querycontext.QueryDataSource;
-import org.apache.iotdb.db.engine.storagegroup.HashVirtualPartitioner;
+import org.apache.iotdb.db.engine.storagegroup.virtualSg.HashVirtualPartitioner;
```

Comment:
The module name is better to be in lowercase.
virtualSg -> virtualsg

Reply:
fixed~